emissionsapi/db.py

```diff
@@ -144,7 +145,50 @@ def get_points(session, polygon=None, begin=None, end=None):
         Carbonmonoxide,
         Carbonmonoxide.geom.ST_X(),
         Carbonmonoxide.geom.ST_Y())
+    return filter_query(query, polygon=polygon, begin=begin, end=end)
 
+
+def get_averages(session, polygon=None, begin=None, end=None):
+    """Get daily averages of all points filtered by time and location.
+
+    :param session: SQL Alchemy Session
+    :type session: sqlalchemy.orm.session.Session
+    :param polygon: Polygon specifying an area in which to search for points.
+                    Defaults to None
+    :type polygon: geoalchemy2.WKTElement, optional
+    :param begin: Get only points after this timestamp, defaults to None
+    :type begin: datetime.datetime, optional
+    :param end: datetime.datetime, defaults to None
+    :type end: Get only points before this timestamp, optional
+    :return: SQLAlchemy Query with tuple of the daily carbon monoxide average,
+             the maximal timestamp the minimal timestamp and the timestamp
+             truncated by day.
+    :rtype: sqlalchemy.orm.query.Query
+    """
+    day = sqlalchemy.func.date_trunc('day', Carbonmonoxide.timestamp)
+    query = session.query(
+        sqlalchemy.func.avg(Carbonmonoxide.value),
+        sqlalchemy.func.max(Carbonmonoxide.timestamp),
+        sqlalchemy.func.min(Carbonmonoxide.timestamp),
+        day).group_by(day)
+    return filter_query(query, polygon=polygon, begin=begin, end=end)
+
+
+def filter_query(query, polygon=None, begin=None, end=None):
+    """Filter query by time and location.
+
+    :param query: SQL Alchemy Query
+    :type query: sqlalchemy.orm.Query
+    :param polygon: Polygon specifying an area in which to search for points.
+                    Defaults to None
+    :type polygon: geoalchemy2.WKTElement, optional
+    :param begin: Get only points after this timestamp, defaults to None
```

Comment:
Actually I would like to leave it that way. Honestly I wrote it that way, because my auto documentation tool writes it that way, but according to that https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format this is how you write it in the sphinx format.

`. Defaults to None.` seems to be the in the google format.

Reply:
Fair enough, but please be consistent. Two lines above, you used `. Defaults to…`